x-pack/plugin/esql/qa/testFixtures/src/main/resources/mapping-dense_vector.json

```diff
@@ -5,7 +5,12 @@
     },
     "vector": {
       "type": "dense_vector",
-      "similarity": "l2_norm"
+      "similarity": "l2_norm",
+      "index_options": {
```

Comment:
ℹ️ @alex-spies It seems that adding explicit index_options is preventing the failure